tests/unit/services/lambda_/test_lambda_utils.py

```diff
@@ -38,113 +35,3 @@ def test_get_handler_file_from_name(self):
         assert "main" == get_handler_file_from_name("main", Runtime.go1_x)
         assert "../handler.py" == get_handler_file_from_name("../handler.execute")
         assert "bootstrap" == get_handler_file_from_name("", Runtime.provided)
-
-
-class TestFilterStreamRecords:
```

Comment:
Why are we removing this entire test class?

Reply:
It's related to the legacy filtering implementation removed in this PR. We have a much more comprehensive aws-validated test suite covering all these cases now